src/components/radios/components/Radio.tsx

```diff
@@ -64,6 +64,7 @@ const Radio: React.FC<RadioProps> = ({
         <input
           onChange={e => {
             setSelected(radioReference);
+            e.persist();
```

Comment:
Do we also need to make this change in line 64 of Box.tsx? https://github.com/NHSDigital/nhsuk-react-components/blob/d9d049aede50bf45876ba6f89bbb4cb5413aa854/src/components/checkboxes/components/Box.tsx#L50-L51

Reply:
Nope, reverted the change :)